magento/magento2
# Move bunch of javascript variables from global scope to local scope in magnifier.js file

### Description (*)
Due to a mistake in [a 9 year old commit](https://github.com/magento/magento2/commit/f4bdb5dfd8641627b74eadcb0d52087a63e67b01#diff-87e191852ca0b0ca8282b27eeca862a28e6a55a65b35fad43eeed33716ea12f8) where a `,` was incorrectly changed to a `;`, a bunch of JS variables became globally scoped and no longer scoped inside the local `Magnify` function.

This PR fixes that.

To better review the changes in this PR, please ignore whitespace changes: https://github.com/magento/magento2/pull/39321/files?w=1

### Related Pull Requests


### Fixed Issues (if relevant)
1. Fixes https://github.com/magento/magento2/issues/36200

### Manual testing scenarios (*)
1. Have a Magento shop with a product with some images
2. Visit that product in the frontend
3. Open your browsers javascript console
4. Execute `console.log(magnifierOptions);`
5. Test the magnifier functionality on the product images

Expected:
4. Shouldn't work, `magnifierOptions` should not be available in global scope
5. Magnifier functionality should keep working the way it worked before

### Questions or comments
I won't be updating/writing automated tests, because I have no idea how to do that in scope of this change. If that's required, please do it yourself :)

### Contribution checklist (*)
